source/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.rst

```diff
@@ -89,6 +91,23 @@ Flags
 
 The command supports the following flags:
 
+.. mc-cmd:: --interactive
+   :optional:
+
+   Offers command line prompts to request the information required to set up a new tenant.
+   Use this instead of the remaining flags when creating a new tenant.
```

Comment:
"Mutually exclusive with all other arguments to this command" ?

Reply:
Yeah, that says it better and more consistent with our language elsewhere.